README.md

```diff
@@ -83,7 +83,7 @@ make clean; make EMACS=/Applications/Emacs.app/Contents/MacOS/Emacs
 
 ### Using MELPA
 
-As explained in the [MELPA guide](https://melpa.org/#/getting-started), is it as easy as typing
+As explained in the [MELPA documentation](https://melpa.org/#/getting-started), it is as easy as typing
 <kbd>M-x package-list-packages RET</kbd> then `r` (**r**efresh the package list), `U` (mark **U**pgradable packages), and `x` (e**x**ecute the installs and deletions).
```

Comment:
This updates all installed melpa packages. This should at least be said, and maybe a way to update only pg?

Reply:
Hi @Matafou, I've just seen your remark. I will force-push a new version very soon.